fahrschein-http-test-support/src/main/java/org/zalando/fahrschein/http/test/AbstractRequestFactoryTest.java

```diff
@@ -103,6 +104,35 @@ public void testGzippedRequestBody() throws IOException {
         assertEquals(responseBody, actualResponse);
     }
 
+    @Test
+    public void testZStandardRequestBody() throws IOException {
```

Comment:
Maybe extract the duplicated code from `testGzippedRequestBody` into a private method.